apps/dotcom/src/utils/iFrame.ts

```diff
@@ -2,19 +2,15 @@ export const isInIframe = () => {
 	return typeof window !== 'undefined' && (window !== window.top || window.self !== window.parent)
 }
 
-export function getTopLevelOrigin() {
-	let origin: string
+export function getParentOrigin() {
 	if (isInIframe()) {
 		const ancestorOrigins = window.location.ancestorOrigins
 		// ancestorOrigins is not supported in Firefox
 		if (ancestorOrigins && ancestorOrigins.length > 0) {
-			origin = ancestorOrigins[ancestorOrigins.length - 1]
+			return ancestorOrigins[0]
```

Comment:
@MitjaBezensek why parent origin vs. top? i could see it being either way honestly - just curious what made you rethink it

Reply:
It was the [comment from Alex](https://github.com/tldraw/tldraw/pull/3192#discussion_r1576503766):

> the parent is more truly the embedder that the top level document, I would guess? e.g. if i embed tldraw in notion then embed notion in my website, notion is still the main embedder i think?

We are probably more interested in who does the initial embedding right? Makes it more actionable for us, we can reach out directly to them.